form-login.js

```diff
@@ -0,0 +1,126 @@
+const emailInput = document.querySelector('input[name="userName"]');
+const passwordInput = document.querySelector('input[name="userPassword"]');
+const loginButton = document.getElementById('loginButton');
+
+const emailError = document.getElementById('emailError');
+const passwordError = document.getElementById('passwordError');
+
+function validateEmail() {
+  const email = emailInput.value.trim();
+  if (!email) {
+    emailError.textContent = '이메일을 입력해주세요.';
+    emailInput.classList.add('error');
+    return false;
+  } else if (!/^[\w.-]+@[\w.-]+\.\w+$/.test(email)) {
+    emailError.textContent = '잘못된 이메일 형식입니다.';
+    emailInput.classList.add('error');
+    return false;
+  } else {
+    emailError.textContent = '';
+    emailInput.classList.remove('error');
+    return true;
+  }
+}
+
+function validatePassword() {
+  const password = passwordInput.value.trim();
+  if (!password) {
+    passwordError.textContent = '비밀번호를 입력해주세요.';
+    passwordInput.classList.add('error');
+    return false;
+  } else if (password.length < 8) {
+    passwordError.textContent = '비밀번호를 8자 이상 입력해주세요.';
+    passwordInput.classList.add('error');
+    return false;
+  } else {
+    passwordError.textContent = '';
+    passwordInput.classList.remove('error');
+    return true;
+  }
+}
+
+function checkFormValidity() {
+  const isEmailValid = validateEmail();
+  const isPasswordValid = validatePassword();
+  loginButton.disabled = !(isEmailValid && isPasswordValid);
```

Comment:
```suggestion
  loginButton.disabled = !isEmailValid || !isPasswordValid;
```

이런식으로도 작성 가능합니다. 둘 중 가독성 좋은걸로 하면 좋습니다.